addons/mail/static/src/models/notification/notification.js

```diff
@@ -80,6 +118,18 @@ registerModel({
     },
     fields: {
         failure_type: attr(),
+        /**
+         * States the classname of the icon for this notification.
+         */
+        iconClass: attr({
+            compute: '_computeIconClass',
+        }),
+        /**
+         * States the text to display as title for this notification.
+         */
+        iconTitle: attr({
+            compute: '_computeIconTitle',
+        }),
```

Comment:
Both are "determines" as changing the value of the field has a direct impact on the displayed value.